.github/workflows/build.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         with:
           repository: jspecify/jspecify
           path: jspecify
+          ref: gradle-9.0.0
```

Comment:
Without this, CI is failing with

````
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':jspecify'.
> Failed to notify project evaluation listener.
   > org.gradle.api.plugins.internal.DefaultDecoratedConvention
````

It seems rather odd that gradle doesn't play well together with an not much older version of gradle used in that other project.

Reply:
No, still fails. Can't reproduce locally yet.

Reply:
I looked at the wrong branch. The issue was that the `samples-google-prototype-eisop` further down was using Gradle 8.9. https://github.com/jspecify/jspecify/pull/768 fixed this and now this PR works.

I think it works even without https://github.com/eisop/jspecify-reference-checker/pull/16 because the gradle-9.0.0 branch was made before the recent changes in jspecify/jspecify.